src/c21client/client.py

```diff
@@ -34,7 +34,12 @@ def get_job(self):
     def send_job_results(self, job_id, job_result):
         endpoint = f'{self.url}/put_results'
         client_id = self.client_id
-        data = {'results': {job_id: job_result}, 'client_id': client_id}
+        agency_id = job_result['data']['agencyId']
+        docket_id = job_result['data']['id']
+        data = {'client_id': client_id,
+                'directory': f'{agency_id}/{docket_id}',
```

Comment:
Should include the file name: `f'{agency_id}/{docket_id}'/{docket_id}.json`

Reply:
Went ahead and fixed this.  Check latest commit for changes!